contributors.md

```diff
@@ -291,19 +362,34 @@ I am an associate member of the Chartered Institute of Personnel and Development
 
 ### Paul Owoicho
 
-* GitHub id: TBA
+* Role: Google Season of Doc: Technical Writer, [OLS project lead](https://github.com/alan-turing-institute/the-turing-way/tree/master/open-life-science-mentoring): Embedding Accessibility in The Turing Way (2020)
+* GitHub id: [paulowoicho](http://github.com/paulowoicho)
 * ORCID: TBA
-* Short bio: TBA
+* Short bio: 
+> I am a Technical Writer / Google Season of Docs (GSoD) Participant working to make The Turing Way consistent, sustainable, and accessible.
+> I have a BSc in Software Engineering from the American University of Nigeria, and an MSc in Data Science from the University of Glasgow.
+> Before now, I worked as a Research Analyst in the Fintech & Innovation Division of Guaranty Trust Bank, Nigeria.
+> While there, I helped to drive the Bank's push to become a platform by creating innovative digital products.
 >TBA
 
 * Personal highlights:
 >TBA
```

Comment:
```suggestion
> The Turing Way is my first foray into open source and has been a fantastic learning experience.
> Not only have I gained a deeper understanding and appreciation for how GitHub works, but I am also learning to prioritise sustainability and empowerment in the work that I do.
> I am also very grateful for the opportunity to work with people (who I may not have met otherwise) from around the world.
```